examples/mrc/README.md

```diff
@@ -20,6 +23,13 @@ python examples/mrc/run_mrc.py --model_name_or_path xlm-roberta-large \
        --warmup_ratio 0.1 --weight_decay 0.1 --save_steps 50000 \
        --overwrite_output_dir --num_train_epochs 1 --evaluation_strategy no
 ```
+This will detect a GPU if present as well as multiple CPU cores for accelerating preprocessing.
+Some hyperparameters (e.g. fp16, batch size, gradient accumulation steps) may need to be changed
+depending on your hardware configuration.
+
+This yields the following results \TODO:
+```add our results here in the HF format as shown in https://github.com/huggingface/transformers/tree/main/examples/pytorch/question-answering
```

Comment:
*** / Aggregate Scores ****
{"avg_passage_f1": 0.7214616561954854, "avg_passage_recall": 0.7060810023084223, "avg_passage_precision": 0.7403183125930369, "avg_minimal_f1": 0.6744985613562953, "avg_minimal_recall": 0.6279638519632065, "avg_minimal_precision": 0.733070067175786}

***** eval metrics *****
  epoch                      =    1.0
  eval_avg_minimal_f1        = 0.6745
  eval_avg_minimal_precision = 0.7331
  eval_avg_minimal_recall    =  0.628
  eval_avg_passage_f1        = 0.7215
  eval_avg_passage_precision = 0.7403
  eval_avg_passage_recall    = 0.7061
  eval_samples               = 238831

Reply:
Added it. thx